lib/tests/test_git.rs

```diff
@@ -1566,7 +1608,20 @@ fn test_export_refs_bookmark_changed() {
         git_id(&new_commit)
     );
     // HEAD should be unchanged since its target bookmark didn't change
-    assert_eq!(git_repo.head().unwrap().name(), Some("refs/heads/feature"));
+    assert_eq!(
+        git_repo
+            .head()
+            .unwrap()
+            .peel_to_commit_in_place()
+            .unwrap()
+            .id(),
+        git_repo
+            .find_reference("refs/heads/feature")
+            .unwrap()
+            .peel_to_commit()
+            .unwrap()
+            .id()
+    );
```

Comment:
Here we need to assert that the `HEAD` is unchanged (i.e. a symbolic ref.)

btw, maybe we can use `git_repo.head_id()`, `.head_name()`, etc.